application/views/js_init/users/js_users.php

```diff
@@ -1,126 +1,178 @@
 <script language="javascript">
-$(document).ready(function() {
+	$(document).ready(function()
+	{
 
-var inbox_master = '<?php echo $this->config->item('inbox_owner_id')[0];?>';
-	
-// Add/Edit Contact
-$('.addpbkcontact, .edit_user').on('click', null, function() {
-if($(this).hasClass('addpbkcontact')) {
-	var user_title = '<?php echo lang('tni_user_add');?>';
-	var type = 'normal';
-	var param1 = '';
-}	
-else if($(this).hasClass('edit_user')) {
-	var user_title = '<?php echo lang('tni_user_edit');?>';
-	var type = 'edit';
-	var param1 = $(this).parents("tr:first").attr("id");
-}
+		var inbox_master = '<?php echo $this->config->item('inbox_owner_id')[0];?>';
 
-$("#users_container").load('<?php echo site_url('users/add_user')?>', { 'type': type, 'param1': param1 }, function() {
-$(this).dialog({
-	title: user_title,
-	modal: true,
-	show: 'fade',
-	hide: 'fade',
-	buttons: {
-	'<?php echo lang('kalkun_save');?>': function() {
-		if($("#addUser").valid()) {
-		$.post("<?php echo site_url('users/add_user_process') ?>", $("#addUser").serialize(), function(data) {
-		$("#users_container").html(data);
-		$("#users_container").dialog({ buttons: { "Okay": function() { $(this).dialog("close"); } } });
-		setTimeout(function() {$("#users_container").dialog('close')} , 1500);
+		// Add/Edit Contact
+		$('.addpbkcontact, .edit_user').on('click', null, function()
+		{
+			if ($(this).hasClass('addpbkcontact'))
+			{
+				var user_title = '<?php echo lang('tni_user_add');?>';
+				var type = 'normal';
+				var param1 = '';
+			}
+			else if ($(this).hasClass('edit_user'))
+			{
+				var user_title = '<?php echo lang('tni_user_edit');?>';
+				var type = 'edit';
+				var param1 = $(this).parents("tr:first").attr("id");
+			}
+
+			$("#users_container").load('<?php echo site_url('users/add_user')?>',
+			{
+				'type': type,
+				'param1': param1
+			}, function()
+			{
+				$(this).dialog(
+				{
+					title: user_title,
+					modal: true,
+					show: 'fade',
+					hide: 'fade',
+					buttons:
+					{
+						'<?php echo lang('kalkun_save');?>': function()
+						{
+							if ($("#addUser").valid())
+							{
+								$.post("<?php echo site_url('users/add_user_process') ?>", $("#addUser").serialize(), function(data)
+								{
+									$("#users_container").html(data);
+									$("#users_container").dialog(
+									{
+										buttons:
+										{
+											"Okay": function()
+											{
+												$(this).dialog("close");
+											}
+										}
```

Comment:
something wrong here?

Reply:
Slide to the right, the code is highly indented here